chain/default_store_test.go

```diff
@@ -16,50 +16,52 @@ import (
 	"github.com/filecoin-project/go-filecoin/proofs"
 	"github.com/filecoin-project/go-filecoin/repo"
 	th "github.com/filecoin-project/go-filecoin/testhelpers"
-	tf "github.com/filecoin-project/go-filecoin/testhelpers/testflags"
```

Comment:
BLOCKING: We will still need flags on all of these tests, if they can be ran in parallel then the `tf.UnitTest(t)` flag is appropriate, else they will need to continue using the `tf.BadUnitTestWithSideEffects(tf)` flag.

Reply:
Thanks @frrist.  Sorry for missing that @mslipper

Reply:
Done.

Reply:
At first pass I didn't realize this was going to allow us to remove the `BadUnitTestWithSideEffects` flag. This is massive, thank you @mslipper!!